build/test-requirements.txt

```diff
@@ -6,4 +6,4 @@ pillow>=8.3.1,<9.1.0
 pytest-benchmark
 pytest-xdist
 wheel
-numpy<1.23.0
+numpy
```

Comment:
Remove this? setup.py should cover it

Reply:
I don't understand - we want the tests to run on the most recent version of numpy.

Reply:
Right. There is no upper bound on version constraint on numpy.

This numpy was not there before I added the `<1.23.0` and we still hit that error. So I think removing this should be fine.